cmd/aws-janitor/main.go

```diff
@@ -143,3 +183,24 @@ func markAndSweep(opts resources.Options, region string) error {
 
```

Comment:
no action required in this change, but in the future it might be worth reporting how many resources were deleted, too (see line directly above this one, I just can't comment on it directly)

(there's probably a similar count obtainable from `CleanAll()`)

Reply:
thanks for your review and feedback Jeff

Added a metric to report the sweep, the `CleanAll()`  function does not return how many resources were deleted, but i think we can add that, i can do that if you think that will be good

ptal

Reply:
yeah, we'd need to pass that information back from `CleanAll()`, but that can happen separately.